projects/cardiovascular-patient-app/src/app/report-bp/report-bp.component.html

```diff
@@ -1,5 +1,10 @@
 <mat-card>
   <mat-card-content>
+    <mat-card class="card-widget">
+      <div>
+        Last BP reported was {{ lastReportedBPdata?.systolic?.value }}/{{ lastReportedBPdata?.diastolic?.value }} on {{ lastReportedBPdata?.systolic?.date }}
+      </div>
+    </mat-card>
```

Comment:
did you test this with a patient who has no prior BP?

Reply:
yes, colin I have tested for the patient(patient ID:- 26247)  There is no prior BP data for the patient.

Reply:
what does it display for that patient?